docs/contributing/feature-stages.md

```diff
@@ -9,37 +9,37 @@ If you encounter an issue with any Coder feature, please submit a
 
 ## Early access features
 
+- **Stable**: No
+- **Production-ready**: No
+- **Support**: GitHub issues
+
 Early access features are neither feature-complete nor stable. We do not
 recommend using early access features in production deployments.
 
 Coder releases early access features behind an “unsafe” experiment, where
```

Comment:
```suggestion
Coder often releases early access features behind an “unsafe” experiment, where
```

We also sometimes have early access via separate executable which is opt-in, such as the Coder Desktop binary or a module/helm install. So it is not exclusively coupled to experiments